channeldb/graph.go

```diff
@@ -447,6 +453,18 @@ func (c *ChannelGraph) AddChannelEdge(edge *ChannelEdgeInfo) error {
 			return err
 		}
 
+		// Mark edge policies for both sides as unknown. This is to
+		// enable efficient incoming channel lookup for a node.
+		for _, key := range([]*[33]byte {&edge.NodeKey1Bytes, 
+			&edge.NodeKey2Bytes}) {
+			
+			if err := putChanEdgePolicyUnknown(edges, edge.ChannelID, 
+				key[:]); err != nil {
+
```

Comment:
remove newline

Reply:
The newline is to separate the wrapped if condition from the code block. This is formatting is used everywhere through out the code base?

Reply:
To separate long lines like this, the first thing would be 
```go
err := putCha...
if err != nil {
    return err 
}
```